src/react/components/pages/modelCompose/modelCompose.tsx

```diff
@@ -784,7 +784,14 @@ export default class ModelComposePage extends React.Component<IModelComposePageP
                 this.setState({
                     isComposing: false,
                 })
-                throw new AppError(ErrorCode.ModelNotFound, error.message);
+                if( ErrorCode.ModelNotFound==='modelNotFound'){
```

Comment:
question: this will always be true because ErrorCode.ModelNotFound is indeed "modelNotFound", should we check `error.name`, `error.errorCode`, or similar property?

Reply:
You're right, I'll fix it.